addons/sale_project/static/tests/tours/sale_project_tour.js

```diff
@@ -0,0 +1,124 @@
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { stepUtils } from "@web_tour/tour_service/tour_utils";
+import { markup } from "@odoo/owl";
+
+registry.category("web_tour.tours").add("task_create_sol_tour", {
```

Comment:
The filename could have the same name than the tour. 🙂 

`task_create_sol_tour.js`